subworkflows/local/eggnog.nf

```diff
@@ -0,0 +1,42 @@
+//
+// Run eggnog-mapper on called ORFs, first optionally downloading the required databases
+//
+
+include { EGGNOG_DOWNLOAD } from '../../modules/local/eggnog/download'
+include { EGGNOG_MAPPER   } from '../../modules/local/eggnog/mapper'
+include { EGGNOG_SUM      } from '../../modules/local/eggnog/sum'
+
+workflow EGGNOG {
+    take:
+        eggnog_dbpath
+        faa
+        collect_fcs
+
+    main:
+        ch_versions = Channel.empty()
+
```

Comment:
This looks unusual. If I am not mistaken then the typical layout will have the `take:` with same number of indentation like the lines below, i.e.
```suggestion
    take:
    eggnog_dbpath
    faa
    collect_fcs

    main:
    ch_versions = Channel.empty()
```
imho not critical, but it does make cross-pipeline-reading/reviewing easier when standards are kept.
This is the first subworkflow that I am looking at, but that seems to be true also for other subworkflows on the first glance.

Additionally, but this might vary from pipeline to pipeline, depending on preferences, channels have often the prefix `ch_`.

Reply:
I can have a look and go through the subworkflows. No issue with fixing the indentation 👍